### PR TITLE
Semantic slot updates nov2018

### PR DESCRIPTION
**Pull request by bennettclark**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updated semantic slot list and variant logic with new slots and values from designers. The new snapshots are from the color value updates. Also, api files needed to be updated for the changed packages.

It's a lot of files to look through, if you're looking for the files that were modified to make all these new snapshots, you'll want to look at:
ISemanticColors.ts
ISemanticTextColors.ts
theme.ts
_semanticSlots.scss
variants.ts

Also various components were updated to change disabledBodyText to the new disabledBodySubtext slot (disabledBodyText is only for text color, while disabledBodySubtext is used for other disabled elements like borders), per Ben and Sirena's direction.

### CODE REVIEW

**Comment by bennettclark** [comment]
@betrue-final-final - I believe you're familiar with these semantic slot updates, can you please review the screener test for this PR?

**Comment by betrue-final-final** [comment]
It appears that the disabled icon slot is wrong. The disabled check, toggle, should be #a6a6a6 and not #c8c8c8, and the toggle thumb looks like it got lighter in code, but the change was subtle.

**Comment by aneeshack4** [comment]
@bennettclark It looks like there are some regressions in screener - is there a plan to fix these?

**Comment by bennettclark** [comment]
@aneeshack4 @betrue-final-final I'm waiting to hear back from Sirena to confirm that the change of disabledBodyText from #c8c8c8 to #a6a6a6 wasn't a mistake or typo, if she confirms that was the value she was given then I'll be starting a convo with Ben and Sirena to get that sorted out

**Comment by betrue-final-final** [comment]
The text change was correct but the icons and actual check box need to remain #c8c8c8. This was something that Sirena, myself and Janet worked out before.

**Comment by bennettclark** [comment]
@srideshpande @dzearing @aneeshack4 - Ben has signed off on these design changes and all checks have passed, so as soon as you have a chance to take a look and sign off on this PR that would be much appreciated!

**Comment by aneeshack4** [comment]
LGTM @srideshpande @dzearing do y'all have anything to add? I think we need 1 code owner review.

**Comment by srideshpande** [comment]
Approving for the choice group changes.

**Comment by aneeshack4** [comment]
@bennettclark looks like there's a merge conflict in Toggle.styles.ts 
Also this needs code owner review - not sure how to tag OfficeDev/uifabric-calendar so pinging!

File: packages/variants/src/variants.ts [hunk 1]
```diff
@@ -88,24 +88,36 @@ export function getNeutralVariant(theme: IPartialTheme): ITheme {
     bodyText: p.neutralPrimary,
     bodySubtext: p.neutralSecondary,
     bodyDivider: p.neutralQuaternaryAlt,
+    focusBorder: p.neutralSecondary,
```
Comment (dzearing) [inline packages/variants/src/variants.ts]: shouldn't this be focusBorder inner/outer?

Comment (dzearing) [reply packages/variants/src/variants.ts]: at least currently focus borders are in 2 parts

Comment (bennettclark) [reply packages/variants/src/variants.ts]: @betrue-final-final - can you please address this question about focus border?

The slots affected by this PR matches the list of semantic slots that was put together by the designers, that has been triple checked and verified multiple times. (Also, I've heard that the double border is slated to go away soon, but I can't speak directly to that.)

Comment (betrue-final-final) [reply packages/variants/src/variants.ts]: I think this got resolved, but if not I don't think I have an opinion on the method, just the result.

**Pull request merged**